.gitignore

```diff
@@ -10,3 +10,4 @@ yarn-error.log
 build
 coverage
 .env
+.jj/
```

Comment:
Hi @broarr, for which kind of files `.jj` is needed?

Thanks

Reply:
`.jj` is used by Jujutsu, a git-backed VCS. I really enjoy it. It makes rebasing so easy

https://jj-vcs.github.io/jj/latest/